src/components/screens/map-screen/map-screen.tsx

```diff
@@ -275,6 +275,50 @@ class MapScreen extends React.Component<Props, State> {
     this.setState({ loadingUserSettings: false });
   }
 
+  /**
+   * Maps favourite locations to locations 
+   */
+  private mapLocationsFromFavouriteLocations = (favouriteLocations:FavouriteLocation[]):Location[] => {
+    return favouriteLocations.map((element) => {
+      const name = element.name;
+      const coordinates = element.latitude + "," + element.longitude;
+      return { name, coordinates };
```

Comment:
```suggestion
  private mapLocationsFromFavouriteLocations = (favouriteLocations: FavouriteLocation[]) => {
    return favouriteLocations.map((element) => {
      const name = element.name;
      const coordinates = element.latitude + "," + element.longitude;
      return { name, coordinates } as Location;
```
In this case it is better to use ```as Location``` if you want your return to have a specific type, also a space is missing